CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Ferrite.jl contributor guide
+
```

Comment:
Maybe we should here at least add a comment referring to https://github.com/firstcontributions/first-contributions or something similar?

Reply:
Good idea, but perhaps not as the very first thing? I added a suggestions below.